src/mesh.cpp

```diff
@@ -410,8 +343,8 @@ scene::IMeshBuffer* cloneMeshBuffer(scene::IMeshBuffer *mesh_buffer)
 		video::S3DVertex2TCoords *v =
 			(video::S3DVertex2TCoords *) mesh_buffer->getVertices();
 		u16 *indices = mesh_buffer->getIndices();
-		scene::SMeshBufferTangents *cloned_buffer =
-			new scene::SMeshBufferTangents();
+		scene::SMeshBufferLightMap *cloned_buffer =
```

Comment:
Can you briefly explain what this does?
Looks great otherwise.

Reply:
This case clones mesh buffer holding `EVT_2TCOORDS` vertices. The old code erroneously creates a buffer for `EVT_TANGENTS` vertices instead, that’s what I fix. It would explode otherwise, would it be used anywhere. (It did actually in some my PR where I needed it)

Reply:
Awesome. Thank you.